ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -122,6 +131,23 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 							acmPolicy.Spec.PolicyTemplates[0].ObjDef.Spec.RemediationAction = sFile.RemediationAction
 						}
 					}
+
+					var policyTemplate = policies[output].(utils.AcmPolicy).Spec.PolicyTemplates[0]
+					if sFile.EvaluationInterval.Compliant != utils.UnsetStringValue {
+						if sFile.EvaluationInterval.Compliant != policyTemplate.ObjDef.Spec.EvaluationInterval.Compliant &&
+							policyTemplate.ObjDef.Spec.EvaluationInterval.Compliant != policyGenTemp.Spec.EvaluationInterval.Compliant {
+							return policies, errors.New("Compliant EvaluationInterval conflict for policyName " + sFile.PolicyName)
```

Comment:
A better error message might point at where the conflict is

Reply:
Yes, the error message has been updated.